autogen/agentchat/groupchat.py

```diff
@@ -61,6 +73,20 @@ def select_speaker_msg(self, agents: List[Agent]):
 
     def select_speaker(self, last_speaker: Agent, selector: ConversableAgent):
         """Select the next speaker."""
+        if self.speaker_selection_method.lower() not in self._VALID_SPEAKER_SELECTION_METHODS:
+            raise ValueError(
+                f"GroupChat speaker_selection_method is set to '{self.speaker_selection_method}'. "
+                f"It should be one of {self._VALID_SPEAKER_SELECTION_METHODS} (case insensitive). "
```

Comment:
It says insensitive here (and the list-checking is case insensitive), but the checks later down appear to be case sensitive. Please correct one or the other.